Esterni/PianoDiQualifica/informazioni.tex

```diff
@@ -0,0 +1,15 @@
+\newcommand{\Titolo}{Piano di Qualifica}
+
+\newcommand{\VersioneDocumento}{\versNdP}
+
+\newcommand{\ACapoRedazione}{Francesco De Marchi \newline Lucrezia Gradi}
```

Comment:
Modificare redazione, approvazione e verifica

Reply:
Inoltre nomi in corsivo, sia approvazione, che redazione, che verifica, che distribuzione